src/pages/schemas/schemaTree/schemaTree.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Table, TableCell, TableContainer, TableRow, IconButton, Collapse, Box, Typography } from '@mui/material';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import { Link } from 'react-router-dom';
+import { KeyboardArrowRight } from '@mui/icons-material';
+
+export type NestedData = {
+  name: string;
+  children?: NestedData[];
+  id?: string;
+};
+
+type NestedTableProps = {
+  data: NestedData[];
+};
+
+const NestedTable: React.FC<NestedTableProps> = ({ data }) => {
+  return (
+    <TableContainer>
+      <Table>
+        {data.map((item) => (
+          <NestedTableRow key={item.name} row={item} />
+        ))}
+      </Table>
+    </TableContainer>
+  );
+};
+
+type NestedTableRowProps = {
+  row: NestedData;
+};
+
+const NestedTableRow: React.FC<NestedTableRowProps> = ({ row }) => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      {
+        <>
```

Comment:
too many levels, only the <> is needed once

Reply:
removed